src/sidebar/helpers/view-filter.js

```diff
@@ -75,18 +57,10 @@ class TermFilter {
    * @param {Annotation} ann
    */
   matches(ann) {
-    const checker = this.checker;
-    if (checker.autofalse && checker.autofalse(ann)) {
-      return false;
-    }
-
-    let value = checker.value(ann);
-    if (Array.isArray(value)) {
-      value = value.map(normalize);
-    } else {
-      value = normalize(value);
-    }
-    return checker.match(this.term, value);
+    const matcher = this.matcher;
```

Comment:
Is this merely a convenience assignment?

Reply:
Yes